src/rgw/rgw_admin.cc

```diff
@@ -4382,6 +4464,34 @@ int main(int argc, const char **argv)
           if (compression_type) {
             info.compression_type = *compression_type;
           }
+          if (data_layout_type_specified) {
+            info.data_layout_type = placement_data_layout_type;
+          }
+          if (current_tail_pool) {
+            info.current_tail_pool = *current_tail_pool;
+          }
+          if (tail_pools) {
```

Comment:
@Jeegn-Chen 
Seems we have no way to remove anything from data_tail_pools.
how about call `info.data_tail_pools.clear();` here?